aqt/mediasrv.py

```diff
@@ -59,9 +64,15 @@ def getPort(self):
     def shutdown(self):
         self.server.shutdown()
 
+def createRequestHandler(addonFolder):
+    """RequestHandler factory"""
+    return type("RequestHandler", (RequestHandler, ),
+                {"_addonFolder": addonFolder})
+
```

Comment:
Any particular reason for the introduction of this method over just setting RequestHandler._addonFolder directly?

Reply:
Hmm, no, not really. I guess it felt a bit cleaner, but considering that RequestHandler is only used in the media server, and considering that the add-ons folder is shared among profiles, this really was unnecessary. I'll be happy to change it to setting _addonFolder directly.

(Sorry about earlier. Clicked the wrong button and marked this as resolved rather than submitting the comment)